### DISCUSSION TITLE
Chinese translation of the MASTG

### DISCUSSION THREAD

**Discussion by ansionasc** [OWASP/mastg#2339] [General]
Hi,I have translate MASTG into chinese,now it's available on Github:[https://github.com/ansionasc/owasp-mastg-zhcn](https://github.com/ansionasc/owasp-mastg-zhcn).
Maybe someone can help add the link to the [contributing page](https://mas.owasp.org/contributing/4_Add_new_Language/)?

**Comment by cpholguera** [comment]
Closing since the page linked doesn't seem to have an updated translation. Please reopen if necessary.

**Reply by ansionasc** [reply]
I have already published an updated translation version. Due to the fact that I am translating in my spare time, it may take one to three months after the English version is released to complete and publish a new version.

**Comment by cpholguera** [comment]
@ansionasc thank you very much for the feedback. That looks really great! I've reopened the discussion. We'll find a place to refer to your translation.
